lib/aws/eks.ts

```diff
@@ -484,6 +487,7 @@ export class EksStack {
               "alb.ingress.kubernetes.io/scheme": "internet-facing",
               "alb.ingress.kubernetes.io/security-groups": lbSecurityGroup.securityGroupId,
               "alb.ingress.kubernetes.io/tags": "stack=hyperswitch-lb",
+              "alb.ingress.kubernetes.io/wafv2-acl-arn": waf.waf_arn,
```

Comment:
@NishantJoshi00 This won't be the final load balancer. We'll have another load balancer on top of envoy which is where we have to add the WAF rules.